runtime/frequency/src/lib.rs

```diff
@@ -646,7 +649,7 @@ impl pallet_balances::Config for Runtime {
 	type WeightInfo = weights::pallet_balances::SubstrateWeight<Runtime>;
 	type MaxReserves = BalancesMaxReserves;
 	type ReserveIdentifier = [u8; 8];
-	type MaxHolds = ConstU32<0>;
+	type MaxHolds = ConstU32<1>;
```

Comment:
Curious why this change. I thought we didn't want to allow holds as they are going away.